components/pagination/assets/Pagination.scss

```diff
@@ -0,0 +1,55 @@
+@import "@not-govuk/sass-base";
+
+.hods-pagination {
+  &__summary {
+    font-family: 'Inter UI', sans-serif;
+    @include govuk-font(16);
+    -webkit-font-smoothing: antialiased;
+    @include govuk-responsive-padding(2, "bottom");
+    @include govuk-responsive-padding(2, "top");;
+    float: right;
+  }
+
+  &__list-items {
+    display: inline-block;
+  }
+
+  &__item {
+    display: inline-block;
+    list-style: none;
+  }
+
+  &__link {
```

Comment:
Are you sure you need to do all of this?

i think the default styling for links should be doing most of this for you.

Reply:
Yes. Without adding the various margins/padding, all of the elements appear smooshed together on the page.